### ISSUE TITLE
"content-block"  Style Class

### ISSUE THREAD

**Issue by hirenshah** [leemunroe/responsive-html-email-template#198]
In the footer section, you have:

`<td class="content-block">`

Where is that class defined?

**Comment by leemunroe** [comment]
Good catch. I think this is legacy and we no longer use this. You can safely remove.

**Comment by bastianandre-myenso** [comment]
I guess the same applies to `class="body"` on the <table> in email-inlined.html? :smiley:

**Comment by webketje** [comment]
Carries some semantic value
